### PR TITLE
CODEOWNERS: Remove alwa-noric from bluetooth mesh

### PR DESCRIPTION
**Pull request by alwa-nordic**
Removes alwa-nordic from the following paths:
/tests/bluetooth/bsim_bt/bsim_test_mesh/
/tests/bluetooth/mesh_shell

This is done by adding new definitions for those paths, inheriting the owners of the previously matching rule, but without alwa-nordic.

Signed-off-by: Aleksander Wasaznik <aleksander.wasaznik@nordicsemi.no>

### CODE REVIEW

**Review by carlescufi — DISMISSED** [review]

File: CODEOWNERS
Comment (jhedberg) [inline CODEOWNERS:811]: This is missing @PavelVPV from both entries.

**Review by jhedberg — CHANGES_REQUESTED** [review]

**Review by jori-nordic — DISMISSED** [review]

File: CODEOWNERS
Comment (Balaklaka) [reply CODEOWNERS:811]: The mesh team in Nordic has planed to go through the codeowners and maintainers for mesh after this has been merged to find out who will get responsibilities for this from us.

Comment (jhedberg) [reply CODEOWNERS:811]: I just thought that at a bare minimum these new entries should at least include the Zephyr Mesh maintainer from the start.

Comment (carlescufi) [reply CODEOWNERS:811]: I agree, let's please add @PavelVPV initially so that we have someone there that actually works on mesh, and then you can override that later once you have figured it out internally

Comment (PavelVPV) [reply CODEOWNERS:811]: I'm fine with this

Comment (alwa-nordic) [reply CODEOWNERS:811]: I think this should be done in a separate PR. The commit message explains the operation that was performed to arrive at the result.

**Review by Balaklaka — DISMISSED** [review]

**Review by jori-nordic — DISMISSED** [review]

**Review by Balaklaka — DISMISSED** [review]

**Review by jhedberg — DISMISSED** [review]

**Review by jori-nordic — APPROVED** [review]

**Review by jhedberg — APPROVED** [review]

**Pull request merged**
